src/libstd/io/mod.rs

```diff
@@ -144,6 +144,16 @@
 //! # Ok(())
 //! # }
 //! ```
+//! Note that you cannot use the `?` operator in functions that do not return a `Result` (e.g. `main()`).
```

Comment:
could you add a new `//!` line above this to give it some room to breathe?

also, this should be `main`, not `main()`, and `Result<T, E>` not `Result`